chains.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "Stakers Hub",
+  "chains": [
+    {
+    "name": "decentr",
+     "address": "decentrvaloper1f4kmcewkr5jatxu7gm2em6hqst2ujg8ec9p5xn",
+    "restake": {
+    "address": "decentr1709spkwudstcnlpvt6cpwhh6lylxxylym9h60j",
+    "run_time": [
+      "every 1 hour",      
+    ],
+    "minimum_reward": 1000000
+        }
+     },
+    {
+     "name": "lumenx",
+     "address": "lumenvaloper12z56ln2vxwhr7r3vra3pfp8r9fk88kfpd3ttpf",
+     "restake": {
+     "address": "lumen1709spkwudstcnlpvt6cpwhh6lylxxyly97499e",
+     "run_time": [
```

Comment:
interval runtimes shouldn't be in an array (that's just for specifying multiple runtimes).

e.g. `"run_time": "every 1 hour"`